example-projects/game-processor/scenario.yaml

```diff
@@ -1,11 +1,42 @@
-version: 0.0.1
-plot-1:
-  eps: 1
-  priority: 2
-  payload:
-    test:
-      subtest: '2'
-  parameters:
-    serverId: '1'
-    playerId: '5'
-    itemId: '3'
+version: "0.0.1"
+user-logs-on:
+  game/server/{serverId}/events/player/{playerId}/connect:
+    playerId :
+      min: 0
+      max: 2000
+    serverId:
+      min: 0
+      max: 4
+user-gameLoop:
+  loop:
+    interval:
+      600
+    cycles:
+      5
+    game/server/{serverId}/events/player/{playerId}/hit:
+      serverId: '1'
+      playerId:
+        regex: '^[\w\d]{1,22}$'
+      payload:
+        crit: 125
+        apDamage: 30
```

Comment:
Why are you generating payloads that are incompatible with the specified message payload structure? 🤔 

https://github.com/asyncapi/simulator/blob/dc8d236869b619347cc312035462a27f73ca47e3/example-projects/game-processor/AsyncAPI/components/messages/PlayerHit.yaml